### DISCUSSION TITLE
Add ARM64 docker image

### DISCUSSION THREAD

**Discussion by lucasiscovici** [langfuse/langfuse#2397] [Ideas]
### Describe the feature or potential improvement

A docker image compatible for the platform arm64

**Comment by marcklingen** [comment]
this'd be helpful, open for contributions on this: https://github.com/langfuse/langfuse/issues/1954

**Comment by marcklingen** [comment]
First version to include arm image was just released: https://github.com/langfuse/langfuse/releases/tag/v2.58.0
Thanks again @rennokki for the contribution
